vision/product-search/cloud-client/src/main/java/com/example/vision/ImportProductSets.java

```diff
@@ -60,7 +57,7 @@ public static void importProductSets(String projectId, String computeRegion, Str
     ProductSearchClient client = ProductSearchClient.create();
```

Comment:
I'm noticing this is leftover from the beta. 
Should we be using the autoclosing methods for the client.
```
try (ProductSearchClient client = ProductSearchClient.create()) {
   // All the code
   // No need to call client.close() now. 
}

Reply:
Wasn't sure if ProductSearch is doing stuff a little differently or not. :)